### PR TITLE
Django 1.9/1.10 compatibility

### PR DESCRIPTION
**Pull request by lucacorti**
Removed SubfieldBase, added AppConfig, fixed tests.

### CODE REVIEW

**Comment by lucacorti** [comment]
This moves compatibility to 1.8/1.9/1.10b which are the currently supported versions of Django. Tests are fine on all supported python versions.

**Comment by josemariaruiz** [comment]
Please @philippbosch, merge this pull request as the current code is breaking the package when using  Django 1.10.

**Comment by salexkidd** [comment]
+1

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
   - 3.5
 env:
-  - DJANGO=1.4.22
-  - DJANGO=1.5.12
-  - DJANGO=1.6.11
-  - DJANGO=1.7.11
-  - DJANGO=1.8.13
+  - DJANGO=1.8
+  - DJANGO=1.9
+  - DJANGO=1.10b1
```
Comment (salexkidd) [inline .travis.yml]: 1.10 released!  😄🍻😄

**Comment by salexkidd** [comment]
@lucacorti LGTM!! @philippbosch san. Please look this pull request

**Comment by philippbosch** [comment]
Just released 0.3.0 with this PR merged in. Thanks so much, @lucacorti!

**Comment by lucacorti** [comment]
\o/

**Pull request merged**